classes/helper.php

```diff
@@ -98,11 +98,14 @@ public static function enable_notifications() : string {
         if (get_capability_info('local/csp:seenotifications')) {
             $cansee = has_capability('local/csp:seenotifications', $PAGE->context, $USER->id);
         }
-        if (!$cansee | ($conf->notifications_enable_enforced + $conf->notifications_enable_reported == 0)) {
+        $notificationsenableenforced = !isset($conf->notifications_enable_enforced) ? 0 : 1;
```

Comment:
isn't this wrong? is could be set, but set to 0

I think it should be empty()

Reply:
Yep, still a WiP. I noticed that as well. Adding WiP to the title didn't stop you from being able to merge it like gitlab does.

Reply:
Fixed.